lib/epi_locator/tr/tr_client.ex

```diff
@@ -61,7 +61,7 @@ defmodule EpiLocator.TRClient do
   end
 
   @impl EpiLocator.TRClientBehaviour
-  def person_search_results(uri), do: search_results(uri, "{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPage")
+  def person_search_results(uri), do: search_results(uri, "{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPageV3")
```

Comment:
What's this syntax with the curly brackets about? Clearly it's been there for a while, it just seems weird to me.

Reply:
It's some XML thing. I'm not sure what it's called.